pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter_test.go

```diff
@@ -1907,6 +1907,39 @@ var _ = Describe("Validating VMICreate Admitter", func() {
 			Expect(causes[0].Field).To(Equal("fake.domain.devices.networkInterfaceMultiqueue"))
 		})
 
+		Context("VMI with network multiqueue", func() {
+			var vmi *v1.VirtualMachineInstance
+
+			BeforeEach(func() {
+				_true := true
+				vmi = api.NewMinimalVMI("testvm")
+				vmi.Spec.Domain.Devices.NetworkInterfaceMultiQueue = &_true
+			})
+
+			It("should accept vmi with autoattached interfaces as the default", func() {
+				causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec, config)
+				Expect(causes).To(BeEmpty())
+			})
+
+			It("should accept vmi with autoattached interfaces configured", func() {
+				_true := true
+				vmi.Spec.Domain.Devices.AutoattachPodInterface = &_true
+				causes := ValidateVirtualMachineInstanceSpec(k8sfield.NewPath("fake"), &vmi.Spec, config)
+				Expect(causes).To(BeEmpty())
+			})
+		})
+
+		It("should reject vmi with a network multiqueue, autoAttach disabled, and without nics", func() {
```

Comment:
Please add another case.
"should reject vmi with a network multiqueue, and non-virio nic". In this case, don't disable `autoAttach`, just don't set it.

Reply:
This one isn't it ?
https://github.com/kubevirt/kubevirt/pull/7432/files/ecf08e1d0bdd4e5c34de550609e6cc7cf83f7ac0#diff-d6f277d0d0db39598f4a419c5e6ac07b4a096a569d94f54d21da04da87d0d44cR1897

Reply:
Yes, ignore my comment.